src/kubeobjects/affinity.go

```diff
@@ -15,6 +17,17 @@ func AffinityNodeRequirement() []corev1.NodeSelectorRequirement {
 	return affinityNodeRequirementsForArches(amd64)
 }
 
+func TolerationMatchingNodeAffinity() []corev1.Toleration {
```

Comment:
```suggestion
func TolerationForAmd() []corev1.Toleration {
```

Seems more intuative imo